### ISSUE TITLE
fine-tuned result of aokvqa

### ISSUE THREAD

**Issue by Richar-Du** [salesforce/LAVIS#52]
I want to fine-tune BLIP on aokvqa, and I download your fine-tuned checkpoint and directly evaluated on the validation set, the result is 50.22. However, when I fine-tune the model myself, the result is 41.89. I didn't change the hyperparameter in the config. Could you provide the hyperparameter that you fine-tune the BLIP on aokvqa?

By the way, did you use other vqa datasets to continue pre-train BLIP before fine-tuning it on the aokvqa?

**Comment by dxli94** [comment]
Hi,

50.22 was obtained by resuming from VQAv2 finetuning. If you directly finetune from pre-trained checkpoint, 41.89 sounds reasonable to me.

Thanks.

**Comment by Richar-Du** [comment]
Great! Thanks for your reply :)
